src/Adapter/DeliveryOptions/AbstractDeliveryOptionsAdapter.php

```diff
@@ -0,0 +1,179 @@
+<?php declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\src\Adapter\DeliveryOptions;
+
+use MyParcelNL\Sdk\src\Model\Consignment\AbstractConsignment;
+
+abstract class AbstractDeliveryOptionsAdapter
```

Comment:
Even een paar regels noemen over de adapters in de readme.md. Met voorbeeld inclusief de try catch uit wc of magento. Uitleggen met die try catch dat je ook een eigen adapter kan schrijven (en waarom).